internal/controller/status.go

```diff
@@ -523,3 +524,14 @@ func setVRGConditionTypeVolSyncPVRestoreError(conditions *[]metav1.Condition, ob
 		Message:            message,
 	})
 }
+
+// set condition when a PVC conflict is detected on the secondary cluster
+func newVRGAsDataProtectedConditionPVCConflict(observedGeneration int64, message string) *metav1.Condition {
+	return &metav1.Condition{
+		Type:               VRGConditionReasonDataProtected,
```

Comment:
@ShyamsundarR @BenamarMk is this has to be a VRGConditionTypeDataProtected or VRGConditionTypeClusterDataReady?

Note: i am using wrong type, i have to correct this place.